EmotiBit/ofxEmotiBit
# Feat lsl 1.14.0

# Description
- Adds nee lib files for LSL 1.14.0

# Changes
- Updated the lib files for Windows, macOS and linux
- Updated LSL workflow to conform to new LSL requirements
- Updated EmotiBit Oscillosocpe to conform to new ofxLSL requirements

# Requirements
- https://github.com/EmotiBit/ofxLSL/pull/7
- https://github.com/EmotiBit/EmotiBit_XPlat_Utils/pull/20


# Issues referenced
- closes #129 
- closes #131 
- closes #136 

# To test
- check out the appropriate branches listed above in requirements
- Make a change in the `emotibitComms.json` file, to tweak the lsl marker stream name/ source_id
  - make sure to replicate the change in [`ofxLSL/marker_test/py/marker.py`](https://github.com/EmotiBit/ofxLSL/blob/a9db0c127eb2785bbab6c35f600c03a2364fb8dd/markerTest/py/marker.py#L20-L21)
- run the marker generator in anaconda
- test the status bar in the Oscillosocpe to reflect the change

# Known issues
- If connecting to a marker stream, where only the name is specified, and not the source_id, if the marker stream crashes, it also crashes the oscilloscope. To prevent an oscilloscope crash, both marker name and source_id should be specified in the emotibitComms file.
